mizar/common/rpc.py

```diff
@@ -236,7 +257,7 @@ def load_transit_agent_xdp(self, veth_peer):
         logger.info(
```

Comment:
Do we need the retry here as well?

Reply:
No as this one is actually unused in the operator, after we transitioned to loading the agent in the interface_service